optimum/bettertransformer/transformation.py

```diff
@@ -79,6 +79,33 @@ def replace_to_bettertransformer(model, config):
     return model
 
 
+def revert_to_original_model(
```

Comment:
```suggestion
def _revert_to_original_model(
```

Reply:
resolving as fixed in [aefc535](https://github.com/huggingface/optimum/pull/676/commits/aefc535d66dc233ca70c71ff40ebb178ad914c0f)